cmd/local/postrun.go

```diff
@@ -36,14 +34,25 @@ func runPostLocal(cmd *cobra.Command, args []string) error {
 	log.Info().Msgf("Kubefirst Console available at: %s", pkg.KubefirstConsoleLocalURLTLS)
 
 	// managing termination signal from the terminal
-	sigs := make(chan os.Signal, 1)
-	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
-	var wg sync.WaitGroup
-	wg.Add(1)
-	go func() {
-		<-sigs
-		wg.Done()
-	}()
+	// todo: handle user inputs (q, ctrl+c, etc)
+	//sigs := make(chan os.Signal, 1)
+	//signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+	//var wg sync.WaitGroup
+	//wg.Add(1)
+	//go func() {
+	//	<-sigs
+	//	wg.Done()
+	//}()
+	//wg.Wait()
+
+	// todo: testing
+	cancelContext()
+	fmt.Println("---debug---")
+	fmt.Println("context killed, waiting...")
+	fmt.Println("---debug---")
```

Comment:
please, prefer to use the log.debug feature to prevent this code ed-up on main, requiring us to fix later in further PR. by testing  already using the debug logger, helps to prevent his clean up prs later.

Reply:
That is cool we have the new logger, that prevents us to send weird console messages to our users. :)

Reply:
![Screenshot 2023-01-20 at 11 00 37](https://user-images.githubusercontent.com/188671/213714987-0755b978-1b54-4ff1-aeda-96c116706787.png)